tests/test_data_ingestion_and_processing.py

```
@@ -182,8 +182,8 @@ def test_mock_orca_parsing(self):
             module = import_module("utils.quantum.orca_parser", "orca_parser")
             self.assertIsNotNone(getattr(module, "parse_orca_output", None))
             logger.info("ORCA parser import successful.")
-        except ImportError as e:
-            logger.error(f"ORCA parser import failed: {e}")
+        except ImportError:
+            logger.error(f"ORCA parser import failed")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)